src/app/core/components/info-card/info-card.component.scss

```diff
@@ -0,0 +1,107 @@
+@import "../../../shared/variables/global.scss";
+
+// Variables
+$card-width: 28rem;
+$card-height: 15rem;
+$mario-logo-width: 200px;
+$mario-logo-height: 200px;
+$box-shadow: 0 8px 10px
+  rgba(
+    $color: #000000,
+    $alpha: 0.45,
+  );
+$letter-spacing-large: 5px;
+$font-size-huge: 110px;
+$letter-spacing-huge: 20px;
```

Comment:
Here you can use rem instead px as weel

Reply:
b5bedb5d13ee14afc035a2620824274cb5e646f3